### PR TITLE
Add GitHub issue templates

### PR DESCRIPTION
**Pull request by stevegrunwell**
Added templates for bug reports and feature requests.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
```
Comment (bswatson) [inline .github/ISSUE_TEMPLATE/bug_report.md]: If you number the templates, they'll show up in the order you want instead of alphabetical.

Comment (bswatson) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Or, could define a config https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser

Comment (stevegrunwell) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Good to know, though I'm not too concerned about ordering here at this point.

Comment (bswatson) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Going to consider this good anyway. Can always update later if needed.

**Pull request merged**